### DISCUSSION TITLE
webGIS Detailsuche

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#93] [General]
in Build 5.23.2801 tritt ein Fehler auf in der Detailsuche mit Auswahllisten.

Falls man den Wert in der Auswahliste overruled, etwas eingibt, das nicht vorgeschlagen wurde .. "geht" die Suche in eine Endlosschleife.

in Build 5.23.2102 kommt dann die Fehlermeldung:
![image](https://github.com/gis-eni/webgis-forum/assets/1842500/6f09eca6-80f5-4d5f-aabb-350dea7f88b5)

**Comment by gis-eni** [comment]
Mit "Endlosschleife" ist gemeint, dass das "Suche läuft Radl" immer weiter läuft, hab ich das Problem gefixt. Danke für den Tipp!
Neuer Build (5.23.2802) kommt heute noch auf den Server.
